{"CVE Code": "CVE-2019-5729", "Description": "Splunk-SDK-Python before 1.6.6 does not properly verify untrusted TLS server certificates, which could result in man-in-the-middle attacks.", "NVD Link": "https://nvd.nist.gov/vuln/detail/CVE-2019-5729", "Vulnerability Categories": "N/A", "Affected Products": "\nSplunk\u00bbSoftware Development Kit\u00bbForPythonVersions  before (<) 1.6.6cpe:2.3:a:splunk:software_development_kit:*:*:*:*:*:python:*:*Matching versions", "EPSS Score": "0.11% Probability of exploitation activity in the next 30 days\n~ 43 % Percentile, the proportion of vulnerabilities that are scored at or less", "EPSS History": "\n #  Date  Old EPSS Score  New EPSS Score  Delta (New - Old) \n 1  2023-03-07  0.89%  0.11%  -0.78 ", "CVSS Scores": "6.8", "CWE": [{"CWE_ID": "CWE-295 Improper Certificate Validation", "CWE_link": "/cwe-details/295/Improper-Certificate-Validation.html"}], "Reference": [{"ref_link": "https://www.splunk.com/view/SP-CAAAQAD", "ref_desc": "Splunk-Python-SDK address one vulnerability | SplunkVendor Advisory", "ref_cve_link": "N/A"}]}